fontforge/python.c

```diff
@@ -7887,12 +7946,20 @@ static PyGetSetDef PyFF_Glyph_getset[] = {
 /*  Glyph Methods  */
 /* ************************************************************************** */
 
-static PyObject *PyFFGlyph_Build(PyObject *self, PyObject *UNUSED(args)) {
+static PyObject *PyFFGlyph_Build(PyObject *self, PyObject *args) {
     SplineChar *sc = ((PyFF_Glyph *) self)->sc;
     int layer = ((PyFF_Glyph *) self)->layer;
+    int accent_hint = false;
+    PyObject *accent_hint_pyo;
```

Comment:
Default assign this to NULL, I'm fairly sure that `PyArg_ParseTuple` doesn't touch it if that optional argument isn't present